package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@trayio/threadneedle",
-  "version": "1.10.3",
+  "version": "1.11.0",
```

Comment:
🤔  branch youre merging into is `threadneedle-1-10-0-release`, but this is 1.11.0, is this intentional?

Reply:
Yh, I noticed the typo yesterday. I'll change the branch name after merging this PR.